src/components/User/index.js

```diff
@@ -11,12 +14,19 @@ const ValidationError = require('../../error/ValidationError');
  */
 async function findAll(req, res, next) {
     try {
-        const users = await UserService.findAll();
-
-        res.status(200).json({
-            data: users,
-        });
+        const verification = jwt.verify(req.token, 'secretkey');
```

Comment:
move to validation middleware like this
`app.get('/books', authenticateJWT, (req, res) => {
    res.json(books);
});`